### ISSUE TITLE
Bat theme looks different than same theme in Helix?

### ISSUE THREAD

**Issue by rahji** [catppuccin/bat#16]
My head is spinning from theme issues between tmux, helix, and bat today. Currently, I'm trying to figure out why the syntax highlighting is different for the same file when viewed through helix and bat. Both are set to Catppuccin Macchiato.

![image](https://github.com/catppuccin/bat/assets/442495/4cbeb859-181a-4fed-8f13-7d8c6d7b675a)

**Comment by backwardspy** [comment]
hey, thanks for raising this!
for helix, are you using the built-in catppuccin theme? it is a few versions behind latest, and so it is missing some of the newer changes to our style guide. for better consistency, consider grabbing the latest theme(s) from https://github.com/catppuccin/helix.

**Comment by rahji** [comment]
Thanks, that was it! They're still a little different but it's just a couple details and not distracting... 

![image](https://github.com/catppuccin/bat/assets/442495/a18f5e90-073f-418e-bd3a-0bc9cee9749f)

**Comment by backwardspy** [comment]
yeah, as far as i know helix does not give us the ability to theme html entities differently from regular text, and the doctype tag takes its colour from the `constant` key which is peach as per our style guide. i believe that's about as close as we can get it right now.

i'm hoping to get the new themes upstreamed soon, but in the mean time i'm glad you're happy with this workaround.

**Comment by ghost** [comment]
i have same exact issue but in kitty terminal

**Comment by uncenter** [comment]
This is the current state of HTML syntax highlighting, compared in Helix, bat, and Neovim:

![Image](https://github.com/user-attachments/assets/6ba64feb-0dfc-446a-a582-099e788679ef)

I've opened https://github.com/helix-editor/helix/issues/13751 to address the gray vs teal brackets. Looking into &nbsp stuff from @rahji's earlier screenshots.

> i have same exact issue but in kitty terminal

This is not related to the terminal, it is the theme for the editor.

**Comment by sud0xenn** [comment]
Add `set-option -sa terminal-overrides ",xterm*:Tc"` to your `.tmux.conf`.
If still doesn't work, add `set -g default-terminal "tmux-256color"` as well.

EDIT: This will fix the background mismatch. I'm still trynna figure out the syntax highlighting mismatch for my tmux and vim setup.

**Comment by sollniss** [comment]
Currently `keyword.operator` is `teal` but according to the style guide it should be `sky`.

Also, `Decorators` and `Annotations` should be `yellow`, not `peach`.
